src/backends/perf_pt/collect.c

```diff
@@ -466,7 +466,6 @@ perf_pt_start_tracer(struct tracer_ctx *tr_ctx, struct perf_pt_trace *trace, str
     // coming from inside the thread. We initialise it to "no errors".
     tr_ctx->tracer_thread_err.typ = 0;
```

Comment:
I think this should be `type = PerfPTCErrorType.Unused` or similar?

Reply:
Ah, yes, it should be `perf_pt_cerror_unused` so as to reference the enum.